### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by BishoyHanyRaafat**
@mark-at-pieces

### CODE REVIEW

File: src/pieces/commands/commands_functions.py [hunk 1]
```diff
@@ -124,8 +124,6 @@ def list_assets(list_type_or_max='assets', **kwargs):
         return
 
     if list_apps:
-        # Logic for listing applications
-        double_space("Listing applications...")
```
Comment (mark-at-pieces) [inline src/pieces/commands/commands_functions.py:128]: any reason why we removed this?

File: src/pieces/commands/commands_functions.py [hunk 2]
```diff
@@ -138,7 +136,6 @@ def list_assets(list_type_or_max='assets', **kwargs):
         else:
             print("Error: The 'Applications' object does not contain an iterable list of applications.")
 
-        print()
```
Comment (mark-at-pieces) [inline src/pieces/commands/commands_functions.py:141]: any reason why we removed this too?

File: src/pieces/commands/commands_functions.py (hunk 1 above)
Comment (BishoyHanyRaafat) [reply src/pieces/commands/commands_functions.py:128]: Consistently
Why all assets models listed without saying "listing ...."

Comment (mark-at-pieces) [reply src/pieces/commands/commands_functions.py:128]: assuming b/c comparing the output in the unit test

Comment (mark-at-pieces) [reply src/pieces/commands/commands_functions.py:128]: as long as this is consistent we are gucci

File: tests/ask_command_test.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import unittest
+from unittest.mock import patch, MagicMock
+from pieces.commands.commands_functions import ask,startup
+import sys
+from io import StringIO
+
+class TestAskCommand(unittest.TestCase):
```
Comment (mark-at-pieces) [inline tests/ask_command_test.py:7]: looking good

File: src/pieces/commands/commands_functions.py (hunk 2 above)
Comment (BishoyHanyRaafat) [reply src/pieces/commands/commands_functions.py:141]: Same thing why there is an extra line at the end of the apps and the rest of the list commands does not have that line?

File: tests/list_command_test.py [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+class TestListCommand(unittest.TestCase):
+    def test_list_command(self):
+        startup()
+        for i in ["models","assets","apps"]:
+            # Redirect stdout to a buffer
+            stdout = sys.stdout
+            sys.stdout = StringIO()
+
+            # Call the function that prints to stdout
+            list_assets(i)
```
Comment (mark-at-pieces) [inline tests/list_command_test.py:17]: not going to lie this is a bit funky that list_assets is the function, but when we can pass in ("models","assets","apps") to list other materials in here as well, I know you didnt write this, but would you mind adding a github issue to split these up into their own functions or to rename this to like a generic list command that will require a material as the first parameter

**Review by mark-at-pieces — COMMENTED** [review]
The tests are looking really solid would you mind adding that GH issue around the list_assets function

**Review by mark-at-pieces — APPROVED** [review]
LGTM, great work added these initial tests

**Pull request merged**